### ISSUE TITLE
snowflake omits a table if it has the same name as another table in a different schema

### ISSUE THREAD

**Issue by chriscardillo** [machow/dbcooper-py#8]
In Snowflake, it is possible for the plain `SHOW TABLES` command to omit tables of the same name in different schemas. For example, if tables `schema_1.users` and `schema_2.users` exist, only one of the two tables will be returned when running `SHOW TABLES`.

Fortunately, if you include `IN DATABASE <db>` in the `SHOW TABLES` command, this problem is alleviated. While it is also possible to use `IN ACCOUNT` to show all tables across all databases, dbcooper-py is currently limited to a single database at a time, so it might make sense to leverage `SHOW TABLES IN DATABASE <db>` to ensure that all tables are returned.

The [`SHOW TABLES` commands in the `_list_tables_sf function`](https://github.com/machow/dbcooper-py/blob/35f6b8e6ffa06e6095cb8b364cdb94c91c3babbc/dbcooper/inspect.py#L83-L90) can could be improved by adding `IN DATABASE` and the database found in the `conn`.

**Comment by machow** [comment]
Thanks @chriscardillo -- any chance you have a link out to the doc, article about this snowflake behavior / bug? Seems like a nice / quick fix!

**Comment by chriscardillo** [comment]
Here is the [Snowflake docs on the `SHOW TABLES` command](https://docs.snowflake.com/en/sql-reference/sql/show-tables.html). While `SHOW TABLES` _does_ work by itself, I don't think it's _intended_ to be used without `IN DATABASE` or `IN ACCOUNT`.

**Comment by machow** [comment]
Hey, the docs specify the default behavior here:

![Screenshot_20220604-140205](https://user-images.githubusercontent.com/2574498/172019934-837f518e-65e4-4e6a-a6e7-d2eee633346b.jpg)

**Comment by chriscardillo** [comment]
I tested this out on my personal Snowflake account, which is maybe a couple of months old, and the behavior above occurs - all tables are returned with just `SHOW TABLES`, even those with the same name.

However, I encountered this issue on my work Snowflake account, which is about 2 years old. When I run dbcooper against this older account, the problem exists, where only one table (out of many with the same name) will show up.

I installed the #9 branch and confirmed that all tables with the same name get turned into accessor functions, in the other account.

I hope this helps clarify!
